.test-infra/jenkins/job_PostCommit_Python_ValidatesContainer_Dataflow.groovy

```diff
@@ -31,5 +31,6 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Py_ValCont',
   // Execute shell command to test Python SDK.
   steps {
     shell('cd ' + commonJobProperties.checkoutDir + ' && bash sdks/python/container/run_validatescontainer.sh')
+    shell('cd ' + commonJobProperties.checkoutDir + ' && bash sdks/python/container/run_validatescontainer_py3.sh')
```

Comment:
Is it possible to run these steps in parallel?

Reply:
Jenkins DSL may have a way to run multiple steps in parallel: https://stackoverflow.com/a/36893393

Reply:
I would suggest spending 30 min to see if it's possible to start this in parallel, and if it works as intended. But can also keep as is.

Reply:
I'm not sure if Gradle build step on each other if we build containers at same time. Probably a better approach is to rewrite run scripts in Gradle since building container is a gradle task and docker has gradle plugin.

Reply:
Ok, we can add a TODO[...] and take care of this later.